service-discovery/README.adoc

```diff
@@ -31,304 +31,399 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 
-You can review all possible commands in https://github.com/redhat-developer/app-services-cli/blob/main/docs/commands/rhoas_cluster.adoc[cli documentation].
+* One application that publishes price updates for a variety of stocks
+* A second application that consumes the price updates for publication on a web page
 
+In addition, you have a Kafka instance in {product}. Each time the first application produces a price update, you want to use the Kafka instance to forward the update as an event to the second, consuming application.
 
-In this Quick Start, you'll connect your OpenShift Streams for Apache Kafka cloud service to your OpenShift cluster using the Red Hat OpenShift Application Services. (RHOAS) Operator and CLI. For more information on Red Hat OpenShift Application Services, please visit link:https://cloud.redhat.com/application-services[cloud.redhat.com.]
+To achieve this behavior, you need a way to connect the applications in OpenShift to your Kafka instance in {product}. Connecting a Kubernetes application to a backing service such as {product} is called __service binding__.
 
-.Prerequisites
-ifndef::community[]
-* You have a Red Hat account.
-endif::[]
-* You have a running Kafka instance in {product}.
+This guide describes how to use the Red Hat OpenShift Application Services (RHOAS) Operator to perform service binding for {product}. The Kubernetes platform used throughout this guide is Red Hat OpenShift.
 
-// Condition out QS-only content so that it doesn't appear in docs.
-// All QS anchor IDs must be in this alternate anchor ID format `[#anchor-id]` because the ascii splitter relies on the other format `[id="anchor-id"]` to generate module files.
-ifdef::qs[]
-[#description]
-Using the {product-long} services in your OpenShift cluster{product-long}.
 
-[#introduction]
-Welcome to the {product-long} Service Discovery quick start. In this quick start, you'll learn how to connect your {product} services with https://quarkus.io/[Quarkus^]. Tutorial will connect Kafka with your Application to produce and consume messages from your Kafka instances in {product}.
-endif::[]
+[id="con-about-service-binding-using-rhoas-operator_{context}"]
+== About service binding using the RHOAS Operator
+
+Connecting a Kubernetes application to a backing service such as {product-long} is called __service binding__. To perform service binding for {product}, you use the Red Hat OpenShift Application Services (RHOAS) Operator.
+
+The RHOAS Operator enables you to expose Kafka instances created in {product} to applications in separate OpenShift clusters. You can use commands in the RHOAS CLI that instruct the RHOAS Operator to inject connection credentials for a specified Kafka instance to an application running in OpenShift. With connection between your application and Kafka instance established, you can then work directly with the Kafka instance using standard OpenShift features and APIs.
```

Comment:
Why "separate" OpenShift clusters? What is separate about them?

Reply:
@pmuir What I had in mind were OpenShift clusters separate from the OSD cluster underpinning the Kafka instance.  But that's probably overthinking it, and the intended nuance will be lost on readers anyway. I'll just go with "applications deployed on OpenShift clusters" :)

Reply:
The fact we run our Kafka instances on OSD(s) is really an implementation detail. It's not secret (we can talk about it in an architecture talk) but I'm not sure it's right to emphasise it in *user* facing docs.

Reply:
ACK, @pmuir  - thanks.